docs/ox_lib/Keybinds/Client.md

```diff
@@ -0,0 +1,44 @@
+# Keybinds
+
+Simple and centralised key mapping management, supporting callbacks when pressing and releasing keybinds.
+
+```lua
+lib.keybinds.new(data)
+```
+* data: `table`
+  * name: `string`
+  * description: `string`
+  * keybind?: `string`
+    * Default: None
+  * disable?: `function`(self: `table`, toggle: `boolean`)
+  * onPressed?: `function`(self: `table`, toggle: `boolean`)
+  * onReleased?: `function`(self: `table`, toggle: `boolean`)
+
+```lua
+local function buttonPressed()
+    print('respects being paid')
+end
+
+local function buttonReleased()
+    print('no longer paying respects')
+end
+
+local keybind = lib.keybinds.new({
+    name = 'respects',
+    description = 'press F to pay respects',
+    keybind = 'F',
+    onPressed = buttonPressed,
+    onReleased = buttonReleased
+})
+```
+
+The keybind object can be retrieved in other scripts by using the getter function.
+```lua
+local keybind = lib.keybinds.get('respects')
+```
```

Comment:
It's an import so the commands are all registered in the invoking resource; you wouldn't be able to use the getter in another script.